mobify/platform-scaffold
# Normalize styles based on 0.15.0 changes

## Changes
- Update component theme styles to account for some of the SDK 0.15.0 changes
- Update theme stylesheets to be consistent, and include the `pw-` prefix for all classes

## How to test-drive this PR
- Run `npm rm -r node_modules/progressive-web-sdk && npm i`
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- **Verify** that styles across the site still look correct
